containers/haskell/.devcontainer/Dockerfile

```
@@ -0,0 +1,32 @@
+# Use the [Choice] comment to indicate option arguments that should appear in VS Code UX
+#
+# [Choice] Haskell version: 8,
+ARG VARIANT=8
```

```suggestion
ARG VARIANT=10
```
I think we can use 8.10

---

> I think we can use 8.10

'ARG VARIANT=8' uses the tag for the docker hub image '8.10.4'. This can either be updated to reflect the repo image or a comment can be added to refer uses to consult the tags used on docker hub.

To confirm build the remote container and run 'ghci --version' which will return 'The Glorious Glasgow Haskell Compilation System, version 8.10.4'.